### DISCUSSION TITLE
Sharing Cookbook by Link

### DISCUSSION THREAD

**Discussion by apkuki** [TandoorRecipes/recipes#2797] [Ideas]
I genuinely like to cook for friends and develop and save my recipes in Tandoor.
So it would be great to share an entire cookbook and all it's recipes I cooked for a night with friend's with them **by link** - the same way I share the single recipes publicly.

It would be cool as not everybody likes to sign into a self hosted system...

**Comment by vabene1111** [comment]
I like the idea, would be a lot of work with the permission system but could be done.

**Reply by csg33k** [reply]
I'd like to add my +1 to this.  That would be an amazingly useful feature.
